### DISCUSSION TITLE
Delete Backup

### DISCUSSION THREAD

**Discussion by daschmidt1994** [nextcloud/all-in-one#4946] [Questions] [answered]
I have all nextcloud data copied from a synology drive. Synology drive makes preview photos in a folder @ea. This I saw to late and backed up my nextcloud with this data. Now I deleted all this folder but in the bakup are the folders. Is in now possible to delete ths backupfolder or break there something?
Or can I run borg init to the folder mapped outside the docker with the borg from the host?

**Comment by szaimen** [comment, marked as answer]
> Is in now possible to delete ths backupfolder?

yes
